lib/auth/fuzz_test.go

```diff
@@ -17,24 +17,11 @@ limitations under the License.
 package auth
 
 import (
-	"encoding/base64"
 	"testing"
 
-	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
 )
 
-func FuzzParseSAMLInResponseTo(f *testing.F) {
```

Comment:
I'm not familiar with this part of the codebase, but are we okay with removing this fuzz test completely? From what I see, it wasn't moved to teleport.e.

Reply:
The reference to this test can be also found in https://github.com/gravitational/teleport/blob/fd4097b93190b6580df793932fe6946aa343b300/fuzz/oss-fuzz-build.sh#L30 

So probably we need to remove it from there.

Reply:
Ah yes. I over-zealously removed this (the always-failing CIFuzz annoys me no end), and forgot to see what I needed to do to move it to the new repo. I'll get onto that.

Reply:
I've added the fuzz test to the enterprise PR. As yet there is nothing to run it as there is no CI job there to do fuzz testing. I can't figure out how the `fuzz/oss-fuzz-build.sh` script is meant to work - I think I'm missing the context in which it runs - there look to be undefined functions and variables, so I suspect OSSFuzz provides this context and corpora. A similar setup will need to be done on the enterprise repo.

@reedloden Is there any feedback you can add?

Reply:
OSS-Fuzz runs `fuzz/oss-fuzz-build.sh` from the config in https://github.com/google/oss-fuzz/tree/master/projects/teleport.

OSS-Fuzz only works on open source projects. We can use https://fuzzbuzz.io/ for `teleport.e`, but it's not configured yet. As long as you ensure Go-native fuzzing is set up (by way of the fuzz test existing), when we do get Fuzzbuzz working, it will Just Work(TM).